src/components/DropDown/DropDown.module.scss

```diff
@@ -1,32 +1,28 @@
 @import '../../styles/variables';
 
 .dropdown {
```

Comment:
In order to allow us to use other Material UI form elements without having to overwrite styles every time, I suggest moving the Material-UI specific files into overrides in the theme file.

To do so, delete all styles here except `width: 100%` and add the following to `src/styles/theme.tsx`:

```js
const materialUiOverrides = {
  // Style sheet name ⚛️
  MuiFormControl: {
    // Name of the rule
    root: {
      // Some CSS
      margin: '10px 0',
      '&:hover': {
        borderColor: colors.white
      }
    }
  },
  MuiFormLabel: {
    root: {
      color: colors.white,
      fontStyle: 'italic',
      '&$focused': {
        color: colors.white
      }
    }
  },
  MuiInputBase: {
    root: {
      color: colors.white,
      fontWeight: 600,
      '&:hover': {
        borderColor: colors.white,
        backgroundColor: colors.grayDark
      }
    }
  },
  MuiOutlinedInput: {
    root: {
      borderRadius: '0',
      '&:hover': {
        borderColor: colors.white
      },
      '&:hover $notchedOutline': {
        borderColor: colors.white
      },
      '&$focused $notchedOutline': {
        borderColor: colors.white
      }
    },
    notchedOutline: {
      borderColor: colors.white
    }
  },
  MuiSelect: {
    icon: {
      color: colors.white,
      fontSize: '2em'
    }
  }
};
```

Now add the following to the bottom of each theme in the same file:

```js
const theme = createMuiTheme({
  ...,
  overrides: { ...materialUiOverrides }
});

const projectorTheme = createMuiTheme({
  ...,
  overrides: { ...materialUiOverrides }
});
```

And finally, add this color at the top: `grayDark: '#444'`